packages/backend/src/open_payments/payment/outgoing/errors.ts

```diff
@@ -46,6 +51,21 @@ export const errorToCode: {
   [OutgoingPaymentError.InvalidReceiver]: 400
 }
 
+export const errorToCode: {
+  [key in OutgoingPaymentError]: string
+} = {
+  [OutgoingPaymentError.UnknownWalletAddress]: GraphQLErrorCode.NotFound,
+  [OutgoingPaymentError.UnknownPayment]: GraphQLErrorCode.NotFound,
+  [OutgoingPaymentError.UnknownQuote]: GraphQLErrorCode.NotFound,
+  [OutgoingPaymentError.WrongState]: GraphQLErrorCode.BadUserInput,
```

Comment:
same suggestion as https://github.com/interledger/rafiki/pull/2768/files#r1633467866

```suggestion
  [OutgoingPaymentError.WrongState]: GraphQLErrorCode.Conflict,
```